gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -25,17 +25,94 @@
     PipelineStepResourceCollection,
 )
 from gnomad_qc.slack_creds import slack_token
+
+# TODO: change to freq_ht when the freq_ht is finalized
+from gnomad_qc.v3.resources.release import release_sites
+from gnomad_qc.v4.create_release.create_release_sites_ht_genomes import (
+    replace_oth_with_remaining,
+)
 from gnomad_qc.v4.resources.annotations import (
     get_combined_frequency,
+    get_freq,
     get_freq_comparison,
 )
 from gnomad_qc.v4.resources.basics import get_logging_path
-from gnomad_qc.v4.resources.release import get_combined_faf_release, release_sites
+from gnomad_qc.v4.resources.release import get_combined_faf_release
 
 logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
 logger = logging.getLogger("compute_combined_faf")
 logger.setLevel(logging.INFO)
 
+faf_pops_to_exclude = POPS_TO_REMOVE_FOR_POPMAX
+
+
+# TODO: not sure if we should put this function in gnomad.utils.annotations
```

Comment:
Lets do that since it seems like Kristen is already using some form of this too. Please put me on the PR.